### DISCUSSION TITLE
Height Problem With Sticky Sidebar

### DISCUSSION THREAD

**Discussion by reasecret** [tailwindlabs/tailwindcss#5066] [Help] [answered]
I'm trying to use a sticky sidebar but sidebar's height is not changing with content.

There is a demo:
https://codepen.io/reasecret/pen/mdmqqxY

How can I fix this?

**Comment by Danster2020** [comment]
If you mean the section with blue background just remove the `h-screen` it prevents it from expanding beyond the size of the viewports height.

**Reply by reasecret** [reply]
If I remove h-screen, sidebar is not staying sticky.

**Comment by ibrahim-kaya** [comment, marked as answer]
`height: max-content;`
Could be your solution

**Reply by reasecret** [reply]
@XpDeviL thanks. That solved this issue. I've used like this:

`<div class="h-screen sticky top-0" style="height: max-content;">`

Looks like Tailwind not supporting max-content with height for now?

**Reply by ibrahim-kaya** [reply]
Not available by default. But you can add it manually by configuring **tailwind.config.js** file.

**Reply by reasecret** [reply]
Like this?
```
module.exports = {
  theme: {
    maxHeight: {
      "content": "max-content"
    }
  }
}
```

**Reply by ibrahim-kaya** [reply]
Yep exactly.

**Reply by reasecret** [reply]
I couldn't use config file but I've added `height: max-content` to style.css file. Thanks...
